ui/packages/catalog-ui-search/src/main/webapp/react-component/location/location.js

```diff
@@ -161,72 +106,13 @@ const LocationInput = props => {
   )
 }
 
-const ddValidators = {
-  lat: value => value <= 90 && value >= -90,
-  lon: value => value <= 180 && value >= -180,
-  north: value => value <= 90 && value >= -90,
-  west: value => value <= 180 && value >= -180,
-  south: value => value <= 90 && value >= -90,
-  east: value => value <= 180 && value >= -180,
-}
-
-let isDms = false
-const dmsValidators = {
-  dmsLat: value => validateInput(value, 'dd°mm\'ss.s"'),
-  dmsLon: value => validateInput(value, 'ddd°mm\'ss.s"'),
-  dmsNorth: value => validateInput(value, 'dd°mm\'ss.s"'),
-  dmsSouth: value => validateInput(value, 'dd°mm\'ss.s"'),
-  dmsWest: value => validateInput(value, 'ddd°mm\'ss.s"'),
-  dmsEast: value => validateInput(value, 'ddd°mm\'ss.s"'),
-}
-
-const getNegOrPosLatLon = (key, value) => {
-  if (value < 0) {
-    return -1 * validLatLon[key]
-  } else {
-    return validLatLon[key]
-  }
-}
-
 module.exports = ({ state, setState, options }) => (
   <LocationInput
     {...state}
     onDraw={options.onDraw}
     setState={setState}
     cursor={key => value => {
```

Comment:
where does this still get called? could we just replace it with `setState` everywhere and eliminate this function?

Reply:
the keyword.js i think calls it?